src/chain_spec.rs

```diff
@@ -981,3 +981,187 @@ pub mod polymesh_itn {
         )
     }
 }
+
+pub mod mainnet {
+    use super::*;
+    use polymesh_runtime_mainnet::{self as rt, constants::time};
+
+    pub type ChainSpec = sc_service::GenericChainSpec<rt::runtime::GenesisConfig>;
+
+    session_keys!();
+
+    fn genesis(
+        initial_authorities: Vec<InitialAuth>,
+        root_key: AccountId,
+        _enable_println: bool,
+        treasury_bridge_lock: BridgeLockId,
+        key_bridge_locks: Vec<BridgeLockId>,
+    ) -> rt::runtime::GenesisConfig {
+        let (identities, stakers, complete_txs) = genesis_processed_data(
+            &initial_authorities,
+            root_key.clone(),
+            treasury_bridge_lock,
+            key_bridge_locks,
+        );
+
+        rt::runtime::GenesisConfig {
+            frame_system: Some(frame(rt::WASM_BINARY)),
+            pallet_asset: Some(asset!()),
+            pallet_checkpoint: Some(checkpoint!()),
+            pallet_identity: Some(pallet_identity::GenesisConfig {
+                identities,
+                ..Default::default()
+            }),
+            pallet_balances: Some(Default::default()),
+            pallet_bridge: Some(pallet_bridge::GenesisConfig {
+                admin: root_key.clone(),
+                creator: root_key.clone(),
+                signatures_required: 3,
+                signers: bridge_signers(),
+                timelock: time::MINUTES * 15,
+                bridge_limit: (100_000_000_000, 365 * time::DAYS),
```

Comment:
This bridge limit also needs to be increased or removed for mainnet.

Reply:
@F-OBrien

Yep - we will be updating the config params as we approach mainnet - this PR just puts in place the mainnet config scaffolding.